components/roadmap/lib.ts

```diff
@@ -16,12 +16,8 @@ interface BinPackOptions {
 }
 
 function getAllRectsWithCollisionsOnXRange (rects: BinPackItem[], x1: number, x2: number) {
-  return rects.filter(rect => {
-    const isLeftOverlapping = rect.left <= x1 && rect.right >= x1
-    const isRightOverlapping = rect.left <= x2 && rect.right >= x2
-    const isWithin = rect.left >= x1 && rect.right <= x2
-    return isLeftOverlapping || isRightOverlapping || isWithin
-  })
+  // this can be calculated as item that are outside the range of x1 and x2.
+  return rects.filter(rect => !(rect.right < x1 || rect.left > x2))
```

Comment:
The one problem I have with this, is that we *may* in the future shrink/expand roadmap items, and this change doesn't support that use case. 

With that said, I'm fine with this change as it covers the bases of our current functionality.

I think we'll want a different, more bin-packing-esque kind of algorithm in the future to consume the least amount of vertical space as possible, and this one may not live long depending on user feedback.